exercises/hamming/canonical-data.json

```diff
@@ -110,14 +110,14 @@
       "property": "distance",
       "strand1": "AATG",
       "strand2": "AAA",
-      "expected": -1
+      "expected": {"error": "leftStrand and rightStrand must be of equal length"}
```

Comment:
```leftStrand``` looks a bit like a Java parameter name. How about 'left strand'? Or 'strands are of different lengths'?